Dev

tokujiTO opened the pull request.

tokujiTO committed:
Merge pull request #29 from fontys-maua-2526-tournament/main

feat: merge

Commit:
Included Page; Included Component

JansenBhodi committed:
US16 - View all teams

US16: Basic layout + axios functionality build up for viewing the teams. Untested with backend. Should be easily adjustable and improvable

Commit:
POST request

JansenBhodi committed:
fix/namingscheme

Naming was not brought up to date during last implement

tokujiTO committed:
Merge pull request #30 from fontys-maua-2526-tournament/create-tournament

Create tournament

tokujiTO committed:
feat: add new custom modal component

tokujiTO committed:
feat: handle close when escape is pressed

muriloorsi committed:
feat: context,provider,services.

yasbonilha committed:
feat: add registerUser page (something is not working well)

JansenBhodi committed:
Updated app, view working

JansenBhodi committed:
Navbar implementation

Navbar now globally appears in the app and does not need to be added separately to each page. Also please use link for navigating

JansenBhodi committed:
tailwind implementation

Finished prettifying of view to be in line with the figma model for the most part.

JansenBhodi committed:
Merge branch 'view-teams' into dev

tokujiTO committed:
fix: useState import in register user page

Commit:
Implemented sidebar

Commit:
Removed import for navbar

JansenBhodi committed:
Merge remote-tracking branch 'origin/navigation-sidebar' into dev

JansenBhodi committed:
bugfixing teamview

the merge for the sidebar introduced bugs for team viewer, fixed them

Commit:
View Tournaments

Commit:
Redesigned list of tournaments

fransanjr committed:
Merge pull request #32 from fontys-maua-2526-tournament/US05-view-tournaments

Us05 view tournaments

Commit:
Renamed CreateTournament; Included route to TournamentCreate

fransanjr committed:
Merge pull request #33 from fontys-maua-2526-tournament/fix/create-tournament

Renamed CreateTournament; Included route to TournamentCreate

tokujiTO committed:
feat: merge with dev

tokujiTO committed:
Merge branch 'dev' into feature/customModal

tokujiTO committed:
feat: standardize components on pages and add toast and eslint to the project

yasbonilha committed:
update code

yasbonilha committed:
feat: add login page

yasbonilha committed:
fix(route): fixed login route

JansenBhodi committed:
prettify team create + guid

prettified team create with the custom fields and buttons.

Added GUID generation for invite code to enforce unique codes for teams

JansenBhodi committed:
Merge branch 'dev' into US#13-Create-Team

tokujiTO committed:
fix: add package-lock.json to gitignore and delete the file

yasbonilha committed:
fix(register user): fixed dob input

yasbonilha committed:
feat: added coachTeamView page

yasbonilha committed:
fix(register user): fixed page's layout and responsiveness

yasbonilha committed:
feat: coachTeamView page

tokujiTO committed:
fix: button reloading page problem

yasbonilha committed:
fix(side bar): visibility on login page

JansenBhodi committed:
Finalized navigation

Fixed the invite as well but its only showing the navigate for some reason

JansenBhodi committed:
field check

Fields need to be filled. this is now reflected but its kinda jank

tokujiTO committed:
Merge branch 'dev' into feature/customModal

tokujiTO committed:
fix: create tournament height

tokujiTO committed:
feat: add required parameter to custom textfield

tokujiTO committed:
Merge pull request #36 from fontys-maua-2526-tournament/feature/customModal

Feature/custom modal

tokujiTO committed:
Merge branch 'dev' into US#13-Create-Team

tokujiTO committed:
Merge pull request #34 from fontys-maua-2526-tournament/US#13-Create-Team

Integrate create team

Commit:
Updated customDateTimePicker, TournamentCreate; Included Toast Notification

Commit:
Implemented Tournament Details

fransanjr reviewed (commented):
We need to consider what is completed before we merge into main.

Commit:
Changed create a tournament title

JansenBhodi reviewed (changes requested):
There are a couple of issues that should be addressed before we push to main, since its supposed to be a fully stable no errors version.

- Tournament create pop-up does not close after successfully creating a tournament
- Tournament view is not reloaded after the create requiring you to wait for to auto refresh
- Tournament view makes a create tournament button for each tournament in view
- the new tournament create buttons lead to an empty page
- Team create pop-up does not close after successfully creating a tournament
- Team view is not reloaded after the create requiring you to wait for to auto refresh

peaceoutommy committed:
Merge branch 'tournament-view' into dev

peaceoutommy committed:
Merged?

peaceoutommy committed:
Refactor Tournament Details and View components; integrate StatusBadge and add cancel functionality

peaceoutommy committed:
Merge pull request #41 from fontys-maua-2526-tournament/US06_CancelTournament

Refactor Tournament Details and View components; integrate StatusBadg…

tokujiTO committed:
fix: [tournament] close automatically at creating and refetch automatically after creating and loading screen implemented

tokujiTO committed:
fix: [team] automatically closes modal after creating and refetch the teams when created successfully

tokujiTO committed:
Merge pull request #42 from fontys-maua-2526-tournament/fix/tournamentCreateAndTeamCreate

Fix/tournament create and team create

yasbonilha committed:
Merge remote-tracking branch 'origin/feature/customModal' into team-leader

tokujiTO committed:
feat: add update team and tournament funciton

tokujiTO committed:
feat: add verification to fields before requesting

Commit:
fix(App): imports and BrowseRouter

JansenBhodi reviewed (approved):
good fixes implemented

Commit:
fix minor things

Commit:
feat: user and team-leader view

tokujiTO committed:
fix: indentation

yasbonilha committed:
fix:integration on coachView

Commit:
Merge branch 'dev' into team-leader

tokujiTO committed:
feat: add user context to the files of the project

tokujiTO committed:
feat: add invitecode and status to update request

peaceoutommy committed:
Implemented basic login and register pages

peaceoutommy committed:
Merge pull request #48 from fontys-maua-2526-tournament/US26_UserRegister

Implemented basic login and register pages

peaceoutommy committed:
is git bugged?

peaceoutommy committed:
Merge pull request #49 from fontys-maua-2526-tournament/US06_CancelTournament

is git bugged?

peaceoutommy committed:
Revert "is git bugged?"

This reverts commit be1837b3d4696c60087fc4a3c4c6907438800397.

peaceoutommy committed:
Revert "Merge pull request #48 from fontys-maua-2526-tournament/US26_UserRegister"

This reverts commit 412ee4a11e24ee05dfadba6b107f6703fd7ec0fb, reversing
changes made to 019481332416c6a541d6355f5ad7d473a8274d1e.

peaceoutommy committed:
Reapply "is git bugged?"

This reverts commit 60e02c51d2f2d6560b98d6d3b70c7f75a7edd87b.

peaceoutommy committed:
Revert "is git bugged?"

This reverts commit be1837b3d4696c60087fc4a3c4c6907438800397.

peaceoutommy committed:
Authentication implementation finish

peaceoutommy committed:
Merge branch 'dev' into US26_UserRegister

peaceoutommy committed:
Merge pull request #50 from fontys-maua-2526-tournament/US26_UserRegister

Authentication implementation finish

peaceoutommy committed:
Add registration functionality and update routing

- Implemented Register component with form handling and API integration.
- Updated App component to include routes for Register and TournamentCreate.
- Replaced CustomButton with a button element for better semantics in TournamentView.

tokujiTO committed:
fix: indentation

tokujiTO committed:
Merge branch 'dev' into team-leader

tokujiTO committed:
Merge pull request #43 from fontys-maua-2526-tournament/team-leader

Team leader

tokujiTO committed:
Merge branch 'dev' into feature/updateTeamAndUpdateTournament

tokujiTO committed:
Merge branch 'dev' into feature/updateTeamAndUpdateTournament

tokujiTO committed:
Merge pull request #51 from fontys-maua-2526-tournament/feature/updateTeamAndUpdateTournament

Feature/update team and update tournament

tokujiTO committed:
fix: app.jsx import error

tokujiTO committed:
feat: fix cors errors and tournament view

tokujiTO committed:
fix: services for standardized request call

tokujiTO committed:
Merge pull request #52 from fontys-maua-2526-tournament/fix/corsProblem

Fix/cors problem

tokujiTO committed:
fix: double uuid dependency

tokujiTO committed:
fix: context provider organization and parameters

tokujiTO committed:
feat: filter to only allow button viewing for people allowed

Commit:
feat: initial development of tournament configuration

tokujiTO committed:
feat: add update and create matches

tokujiTO committed:
feat: better homepage and logos added

tokujiTO committed:
feat: add user to team, list members, delete team, edit team, delete tournament, delete match

tokujiTO committed:
feat: show team names instead of ids on match cards

yasbonilha committed:
Merge remote-tracking branch 'origin/dev' into user-tourn-config

yasbonilha committed:
fix: tournament modal

yasbonilha committed:
fix: removing brackets page

tokujiTO committed:
fix: customDateTimePicker on change

tokujiTO committed:
Merge pull request #53 from fontys-maua-2526-tournament/user-tourn-config

User tourn config

tokujiTO committed:
fix: import errors and context not signing out
